File: test_app/integration_tests/android_mocks/src/main/java/com/voxeet/sdk/services/AudioService.java
package com.voxeet.sdk.services;

import com.voxeet.promise.Promise;
import com.voxeet.sdk.services.audio.LocalAudio;
import com.voxeet.sdk.services.audio.RemoteAudio;

public class AudioService {

    public final LocalAudio local = new LocalAudio();

    public RemoteAudio remote = new RemoteAudio();

    public boolean isSpeakerOn() {
        return false;
    }

    public void setSpeakerMode(boolean isSpeakerOn) {}
}

File: lib/src/sdk_api/audio_service.dart
import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/audio/local_audio.dart';
import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/audio/remote_audio.dart';

/// The AudioService allows changing audio settings for the local and remote participants.
///
/// This service is available in SDK 3.7 and later.
/// {@category Services}
class AudioService {
  /// Audio settings for the local participant.
  final LocalAudio localAudio = LocalAudio();

  /// Audio settings for remote participants.
  final RemoteAudio remoteAudio = RemoteAudio();
}
